sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventsView.java

```diff
@@ -539,95 +539,96 @@ public HorizontalLayout createStatusFilterBar() {
+								grid.bulkActionHandler(items -> {
+
+									EventGrid eventGrid1 = (EventGrid) this.grid;
+									List<EventReferenceDto> references = eventGrid1.asMultiSelect()
+										.getSelectedItems()
+										.stream()
+										.map(EventIndexDto::toReference)
+										.collect(Collectors.toList());
+									if (references.size() == 0) {
+										new Notification(
+											I18nProperties.getString(Strings.headingNoEventsSelected),
+											I18nProperties.getString(Strings.headingNoEventsSelected),
+											Notification.Type.WARNING_MESSAGE,
+											false).show(Page.getCurrent());
+
+										return;
+									}
+
+									ControllerProvider.getDocGenerationController().showEventDocumentDialog(references);
+								});
+							}));
+					}
+				} else if (UserProvider.getCurrent().hasUserRight(UserRight.EVENT_DELETE)) {
 					bulkActions.add(
 						new MenuBarHelper.MenuBarItem(
-							I18nProperties.getCaption(Captions.actionGroupEvent),
-							VaadinIcons.FILE_TREE,
+							I18nProperties.getCaption(Captions.bulkRestore),
+							VaadinIcons.ARROW_BACKWARD,
 							mi -> grid.bulkActionHandler(
-								items -> ControllerProvider.getEventController()
-									.linkAllToGroup(eventGrid.asMultiSelect().getSelectedItems(), eventGrid))));
-				}
-				bulkActions.add(
-					new MenuBarHelper.MenuBarItem(
-						I18nProperties.getCaption(Captions.ExternalSurveillanceToolGateway_send),
-						VaadinIcons.SHARE,
-						mi -> grid.bulkActionHandler(
-							items -> ControllerProvider.getEventController()
-								.sendAllSelectedToExternalSurveillanceTool(eventGrid.asMultiSelect().getSelectedItems(), eventGrid)),
-						UserProvider.getCurrent().hasUserRight(UserRight.EXTERNAL_SURVEILLANCE_SHARE)
-							&& FacadeProvider.getExternalSurveillanceToolFacade().isFeatureEnabled()));
-
-				if (isDocGenerationAllowed()) {
-					bulkActions.add(
-						new MenuBarHelper.MenuBarItem(I18nProperties.getCaption(Captions.bulkActionCreatDocuments), VaadinIcons.FILE_TEXT, mi -> {
-							grid.bulkActionHandler(items -> {
-
-								EventGrid eventGrid1 = (EventGrid) this.grid;
-								List<EventReferenceDto> references = eventGrid1.asMultiSelect()
-									.getSelectedItems()
-									.stream()
-									.map(EventIndexDto::toReference)
-									.collect(Collectors.toList());
-								if (references.size() == 0) {
-									new Notification(
-										I18nProperties.getString(Strings.headingNoEventsSelected),
-										I18nProperties.getString(Strings.headingNoEventsSelected),
-										Notification.Type.WARNING_MESSAGE,
-										false).show(Page.getCurrent());
-
-									return;
-								}
-
-								ControllerProvider.getDocGenerationController().showEventDocumentDialog(references);
-							});
-						}));
+								items -> ControllerProvider.getEventController().restoreSelectedEvents(items, (EventGrid) grid),
+								true)));
```

Comment:
@adinaflorea9 mentioned in the ticket requirement that the change should be done for Event Participants too.
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/c0fff605-2137-4632-b0d8-371a4cd4c5ff)

Reply:
Thanks! Missed that somehow!